Sara/mkt 370 update pricing and billing with usage (#585)
returntocorp/semgrep-docs: docs/semgrep-app/pricing-and-billing.mdx

| Tier | Price | Description |
| ---- | ------- | -------- |
| **Community** | Free | For general-purpose security scanning.  |

> | **Team** | US$40 monthly per developer | For the enforcement of company-specific coding standards, private rules, and the analysis of findings.  |

Reviewer: (Feel free to resolve after you read this comment) Just so I can understand r2c terminology: I guess that there is some business logic decision behind this renaming... just not sure that every user of Semgrep is a developer. "Contributor" looked fine as well. Let me know if there is some thread I can read about it or just ignore this comment. :)

| **Enterprise** | Custom pricing | For custom solutions, including deployment into virtual private clouds (VPCs) with dedicated technical support.  |

Semgrep App integrates with a GitHub or GitLab account at either the organizational or individual level. There is no limit to the number of members in an organization for all tiers.

Billing depends on the number of **developers**. Any organization member who has made a commit in a project within the past 30 days is considered a **developer**.

Members who do **not** make any commits on the code do not need to pay for Semgrep and are not billed. Semgrep Team tier features apply to the entire organization, even if not all members are billed.

### Comparing free and paid tier features

#### Organizational management and usage limits

| Feature  | Community | Team | Enterprise |
| -------  | --------- | ------- | -------- |
| **Member limit** | 20 developers | ♾️  unlimited | ♾️  unlimited |
| **Project limit** |♾️  unlimited | ♾️  unlimited | ♾️  unlimited |

Notes:

* Member usage limits are visible in your [settings page](https://semgrep.dev/orgs/-/settings).
* If your organization exceeds the member limit:

>     * Only the 20 most recent developers can view findings from scans run by Semgrep.

Reviewer: (Again, feel free to ignore:)

```suggestion
    * Only the 20 most recent developers can view findings from Semgrep scans.
```

## Determining your plan needs

### Number of developers

Within your team or organization, assess the number of members that make commits. That determines the number of developers needed for the plan purchase.

> For example, if a project has 4 unique developers who create commits during the billing period and Semgrep scans their repositories, only 4 developers are required even if the organization has a total of 10 members. If these unique developers commit to many projects within the same organization, they will only be counted once, so no additional cost is incurred.

Reviewer: (Again feel free to ignore and resolve) I guess incur meaning is obvious, but it seems to be a rather C2 word (see [dictionary](https://www.merriam-webster.com/dictionary/incur)), probably fine for natives and advanced English speakers. In case you want to simplify it, maybe:
```suggestion
For example, if a project has 4 unique developers who create commits during the billing period while Semgrep is scanning their repositories, only 4 developers are required even if the organization has a total of 10 members. If these unique developers commit to many projects within the same organization, they are counted once, so no additional cost is charged.
```
